backend/src/components/User/model.ts

```diff
@@ -1,7 +1,6 @@
 import * as bcrypt from 'bcryptjs';
```

Comment:
seems like this update can broke current auth-repository from frontend, please update FE accordingly inside this PR